target/scripts/start-mailserver.sh

```diff
@@ -256,9 +256,8 @@ function _defunc
 # ------------------------------------------------------------
 
 _notify 'tasklog' "Welcome to docker-mailserver $(</VERSION)"
-_notify 'inf' 'ENVIRONMENT'
-[[ ${DMS_DEBUG} -eq 1 ]] && printenv
 
+print_environment
```

Comment:
From a logic point of view, `print_environment` should "print environment". The decision if that function output something should not be made within that function, but like it was directly in `start-mailserver.sh`.

Reply:
I see you'r point. I will revert this.

Reply:
Done. [de84818e8389c57fe4ee19929a8a8f7949cbb9c6]